hugegraph-api/src/main/java/com/baidu/hugegraph/api/schema/IndexLabelAPI.java

```diff
@@ -272,6 +274,7 @@ public void checkUpdate() {
             if (this.checkExist != null) {
                 builder.checkExist(this.checkExist);
             }
+            builder.rebuild(this.rebuild);
```

Comment:
prefer keep "Boolean rebuild" define and call builder.rebuild(this.rebuild) if (this. rebuild != null)